.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - '[0-9]*'
+jobs:
+  cockpituous:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/cockpit-project/release
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
+          export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+
+          echo '${{ secrets.SSH_KNOWN_HOSTS }}' > ~/.ssh/known_hosts
```

Comment:
Note that skobyda does not have cockpit's secrets, and they really don't belong to personal repos. If we want to do automated releases, I suggest to move the project to ~cockpit-project first.

An alternative is to use packit right away, it can propose PRs against Fedora dist-gits and we also want it anyway to run gating tests upstream.

Reply:
So drop this workflow and introduce packit.yaml with extra configuration for doing the release somehow from there.

Reply:
I haven't played with this recently, but it's a thought how such "personal" projects could be auto-released. But also, this does not need to block this PR -- the first release to Fedora needs to happen manually anyway, and putting the release tarball on github manually isn't hard either.

Reply:
Removed release workflow. First release will be done manually anyway